stargazer/stargazer.py

```diff
@@ -59,6 +58,16 @@ def validate_input(self):
         else:
             self.dependent_variable = targets[0]
 
+    def new_len(self):
+        """
+        Adding a more informative warning when user don't enter a single model as a list
+        """
+        if isinstance(self.models,list):
```

Comment:
Notice that as of now, other containers (``set``s, ``tuple``s...) would have worked fine. I'm OK with requiring a list, but please add a docstring to ``__init__`` (in the format of ``add_line``'s one) that documents this.